course/layouts/partials/course_content.html

```diff
@@ -1,14 +1,25 @@
-{{ $shouldCollapseTitle := gt (len .Title) 75 }}
+{{ $data := .context }}
+{{ $shouldCollapseTitle := gt (len $data.Title) 75 }}
+
+{{/*  scale images only for instructor insights.   */}}
+{{ if isset . "isInstructorInsightsPage" }}
+  <style>
+    img {
+      width: 100%;
+    }
+  </style>
+{{ end }}
+
```

Comment:
This inline style is not looking much good to me, please let me know if anyone has a more suitable solution.
An alternative solution (possibly but not sure) is mentioned [here](https://github.com/mitodl/ocw-hugo-themes/issues/65#issuecomment-1023112469)

Reply:
@MAbdurrehman12 There is already a stylesheet that is conditionally loaded on Instructor Insights pages.  It's located at `course/assets/css/instructor-insights.scss`.  Here's the way it's loaded:

 - `course/assets/instructor-insights.js` runs `import "./css/instructor-insights.scss"` to ensure it's included in the bundle
 - In `base-theme/assets/webpack/webpack.common.js`, we define an `instructor_insights` webpack bundle that generates `/static/css/instructor_insights.css` and `/static/css/instructor_insights.js`
 - Our Webpack configuration uses an `AssetsPlugin` to write out a file at `base-theme/data/webpack.json` which is a JSON file containing all of the expected URLs to the various assets generated by Webpack
 - In `course/layouts/partials/extrahead.html`you see the following code:
 ```
{{- $instructorInsights := .Site.Data.webpack.instructor_insights -}}
...
{{- if eq $layout "instructor_insights" -}}
<link href="{{ partial "webpack_url.html" $instructorInsights.css }}" rel="stylesheet">
{{- end -}}
 ```
 
When instructor insights pages are generated by `ocw-to-hugo` from legacy courses, they have `layout: instructor_insights` set in their front matter.  This causes Hugo to choose `course/layouts/pages/instructor_insights.html`, but also the `extrahead.html` partial inspects that property and conditionally loads the extra CSS.  Basically, all you should have to do is add this to `instructor-insights.scss`, if your intent is to have all images take up 100% of the width of their parent and scale up:

```
img {
  width: 100%;
}
```

Hopefully this clears things up for you, let me know if you have additional questions

Reply:
Perfect! I tried to look if any conditional loading has already been done but I guess I couldn't figure it out. 
This approach looks good, I'll go on and update the PR.
Thank you for the explanation

Reply:
@gumaerc I reverted the commits in this PR and forced pushed and it got closed :P I guess since there were no file changes so github closed this PR and I'm unable to reopen it so I have created a new [PR](https://github.com/mitodl/ocw-hugo-themes/pull/388) and requested a review from you.